lucene/ivy-versions.properties

```diff
@@ -257,9 +259,7 @@ org.codehaus.janino.version = 3.0.9
 /org.codehaus.janino/commons-compiler = ${org.codehaus.janino.version}
 /org.codehaus.janino/janino = ${org.codehaus.janino.version}
 
-/org.codehaus.woodstox/stax2-api = 3.1.4
-/org.codehaus.woodstox/woodstox-core-asl = 4.4.1
-
+/org.codehaus.woodstox/stax2-api = 4.2
```

Comment:
Weird that the API comes from woodstox but now our impl from "aalto-xml"?  Just want to confirm; it's odd.

Reply:
Yes, it's odd but intended/expected. I tried digging into this more too as I inherited this from Mark.